lib/langchain/tool/base.rb

```diff
@@ -72,6 +72,22 @@ def tool_description
       self.class.const_get(:DESCRIPTION)
     end
 
+    #
+    # Returns the SCHEMA constant of the tool
+    #
+    # @return [Hash] tool description
+    #
+    def tool_schema
+      {
+        name: tool_name,
+        description: tool_description,
```

Comment:
not related to this pr, but maybe rename tool_name and tool_description to just name and description? looks redundant to me the `tool_` prefix. thoughts @andreibondarev ?